Vizzuality/heco-invest
# Let 456 filters

This PR modifies the discovery search filters and autocomplete to work as advanced search

## Testing instructions

As of now, and more evident on the homepage, if user applies filters they’ll be redirected to the discover/search pages right away. This should not happen; the user has to be able to apply filters and still be able to search. 

[Updated filters design](https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=2%3A4807)

## Tracking

[LET-456](https://vizzuality.atlassian.net/browse/LET-456)
